index.Rmd

```diff
@@ -51,6 +51,17 @@ These have not yet been run as live sessions, but you can work through the mater
 
 * [Introduction to Python](https://github.com/moj-analytical-services/intro-to-python)
 * [AWS Tooling demos](https://github.com/moj-analytical-services/mojap-aws-tools-demo)
+* [Pyspark in AP](https://github.com/moj-analytical-services/pyspark-in-AP-guidance)
+* [Prisoid Academy - Python Training](https://github.com/moj-analytical-services/Prisoid-Academy/tree/main/Python)
```

Comment:
I'd maybe add a little detail that this is specific to prison data and you'll need access to Nomis?